### ISSUE TITLE
[BUG] Heroku updated their default support python

### ISSUE THREAD

**Issue by k-r-a-s-s** [andrew-hossack/dash-tools#60]
Hi Andrew, a simple one:

Log from Heroku:
```

-----> Building on the Heroku-22 stack
-----> Using buildpack: heroku/python
-----> Python app detected
-----> Using Python version specified in runtime.txt
 !     Requested runtime 'python-3.8.10' is not available for this stack (heroku-22).
 !     For supported versions, see: https://devcenter.heroku.com/articles/python-support
 !     Push rejected, failed to compile Python app.
 !     Push failed
```

And new heroku supported python versions: https://devcenter.heroku.com/articles/python-support

> "By default, newly created Python apps use the python-3.10.7 runtime. You can also [specify a different supported Python version](https://devcenter.heroku.com/articles/python-runtimes)."

That's it, thanks!

kev

**Comment by andrew-hossack** [comment]
Hey @k-r-a-s-s I'll have an update pushed out in a few hours. For now you can go into runtime.txt and change the python version directly. Thanks for bringing this to my attention!

**Comment by andrew-hossack** [comment]
Problem fixed in v1.8.4

**Comment by k-r-a-s-s** [comment]
amazing, thanks for your work on this - really streamlined my time-to-sharing with my colleagues.
